cloudevents/working-drafts/xml-format.md

```diff
@@ -62,6 +62,11 @@ be expected from an XML document processing perspective.
 Where an event (or batch of events) is represented as a complete XML document,
 an XML document preamble SHOULD be included to ensure deterministic processing.
 
+XML comments are permitted anywhere within an XML representation of a
```

Comment:
@duglin I've modified this in a second commit (only separated from the first so it's easier to spot the differences and change if necessary). I've included the behavior both here and in section 3.3 so it's harder to miss. See what you think.